### ISSUE TITLE
Hierarchy grid should have area-expanded attributes

### ISSUE THREAD

**Issue by Alex-Bubblemaster** [telerik/kendo-ui-core#3369]
### Bug report

Reported in ticket ID 1116245

### Reproduction of the problem

Grouped grid

![image](https://user-images.githubusercontent.com/10980842/27786273-2cd952f2-5fe9-11e7-878c-958733073692.png)

Hierarchy grid

![image](https://user-images.githubusercontent.com/10980842/27786298-4804aaa4-5fe9-11e7-9ec2-ee7b1ef6cbfb.png)

### Current behavior

`area-expanded` attributes are missing

### Expected/desired behavior

`area-expanded` attributes should be added to the `k-hierarchy-cell`

### Environment

* **Kendo UI version:** 2017.2.621
